__tests__/app.test.js

```diff
@@ -136,12 +136,56 @@ describe('/api/articles/:article_id/comments', () => {
                 })
         })
     })
+    describe('POST: Add comments to an article using article_id parametric endpoint', () => {
+        test('POST 201: will post the comment and return with the added comment', () => {
+            const newComment = {
+                username: "rogersop",
+                body: "Hello?! Is this thing on?"
+            }
+            return request(app)
+                .post('/api/articles/6/comments')
+                .send(newComment)
+                .expect(201)
+                .then(({ body }) => {
+                    const { comment } = body
+                    expect(comment.author).toBe("rogersop")
+                    expect(comment.body).toBe("Hello?! Is this thing on?")
+                })
+        })
+        test('POST 400: will fail to post if any fields are empty and return message "Bad Request"', () => {
+            const newComment = {
+                username: "rogersop",
+            }
+            return request(app)
+                .post('/api/articles/6/comments')
+                .send(newComment)
+                .expect(400)
+                .then(({ body }) => {
+                    const { message } = body
+                    expect(message).toBe("Bad request")
+                })
+        })
+        test('POST 404: will fail to post if valid article_id is entered but is non-existent', () => {
+            const newComment = {
+                username: "rogersop",
+                body: "Can anyone hear me?"
+            }
+            return request(app)
+                .post('/api/articles/999/comments')
+                .send(newComment)
+                .expect(404)
+                .then(({ body }) => {
+                    const { message } = body
+                    expect(message).toBe("Not found")
+                })
```

Comment:
We could also consider a couple of other scenarios:
- invalid article_id
- username not found, see what happens when you try to post a comment with a username that doesn't exist on the users table.